sdk/data/aztables/internal/zz_table_client.go

```diff
@@ -68,11 +66,11 @@ func (client *TableClient) createCreateRequest(ctx context.Context, dataServiceV
 		reqQP.Set("$format", string(*queryOptions.Format))
 	}
 	req.Raw().URL.RawQuery = reqQP.Encode()
-	req.Raw().Header["x-ms-version"] = []string{string(client.version)}
+	req.Raw().Header["x-ms-version"] = []string{"2019-02-02"}
 	if options != nil && options.RequestID != nil {
 		req.Raw().Header["x-ms-client-request-id"] = []string{*options.RequestID}
 	}
-	req.Raw().Header["DataServiceVersion"] = []string{string(dataServiceVersion)}
+	req.Raw().Header["DataServiceVersion"] = []string{"3.0"}
```

Comment:
Why doesn't this come from the variable anymore?  Should user be able to configure?

Reply:
This is not user configurable.